### ISSUE TITLE
[1.3.0(27)] Samsung Galaxy Note 2 / 4.4.2- Exporting never ends-Create lomotif with few motifs and add music and export.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-android-issues#266]
Frequency- Every time

Action:
1: Launch the app.
2: Tap on "START" and select few motifs of any category from motif tab.
3: Add music and navigate to "Create" screen.
4: Tap on "Export" option.
5: Check if exporting never ends.

Expected result: The progress on the export progress bar should get completed 100% and user should be navigated to share screen.

Actual result: Exporting progress increases till 15-20 % and again becomes 0 % and it keeps on loading. 

Refer following video for more details:
http://www.screencast.com/t/AaIEfQHSHV

**Comment by hiteshbheda** [comment]
@xiuie I can't reproduce this issue for build [1.3.0 (27)] on Asus Zenfone Go 4.5 / Android 5.1

**Comment by pallavipillai** [comment]
I can't reproduce the issue for build [1.3.0 (27)] on Nexus 6/Android 6.0.1, Moto G4+/Android 6.0.1

**Comment by xiuie** [comment]
@testershub Can your reproduce this on another device please?

**Comment by testershub** [comment]
@xiuie ,No, We are unable to reproduce on another device, It is specifically reproducible on Samsung Galaxy Note 2(4.4.2).

**Comment by testershub** [comment]
@xiuie I can reproduce this issue for build [1.4.6-r2 (38)] on Samsung Galaxy Note 2 / 4.4.2 and Samsung Galaxy S4 / 5.0.1.

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.4.6-r2 (38)] on Asus Zenfone Go / Android 5.1 but for more than 50-60 motifs.

But if i add 20-25 motifs then it is not reproducible
